### DISCUSSION TITLE
Spanish Translation!

### DISCUSSION THREAD

**Discussion by WarriorAmhar** [philote/cthulhudark#12] [General]
[es.json](https://github.com/philote/FoundryVTT-CthulhuDark/files/13605753/es.json)

Hi, I'm not sure if this is the proper way of handling it, but I translated the file to Spanish so I can use it. Lemme know if it works and can be updated with it!

**Comment by philote** [comment]
Thanks for the file. I have added it to version 1.1.1 and released it.

**Reply by WarriorAmhar** [reply]
Hey, sorry but I just realized I kept the tilde and other marks that don't do well in the system and show as an error. This is the version with none of that. Lemme know if it's better. 

I haven't figured out how to make it so they show up, but it's still readable as long as it doesn't show those characters as errors.
Thanks again!! 
[es.json](https://github.com/philote/FoundryVTT-CthulhuDark/files/13607736/es.json)

**Reply by philote** [reply]
I did another release with your new file, its version 1.1.2.

for the special characters, I would think you could use the HTML codes for them. Like maybe these?
https://www.thoughtco.com/html-codes-spanish-characters-4062194

**Reply by philote** [reply]
Oh! also, take a look at the Portuguese (BR) translation. It has some of these characters (though maybe not the one's you need)
https://github.com/philote/FoundryVTT-CthulhuDark/blob/1.1.2/lang/pt-BR.json
